JustFixNYC/who-owns-what
# A/B test Building Alerts vs. Building Updates

testing the following hypotheses: 
- do users read the title text and description of the feature on the buildings page? 
- once we control for the CTA that has a higher rate of clicks, will we see a significant change in the number of conversions based on the "building updates" vs. "building alerts" framing? 

A: master
- CTA: Follow Building (previously Add Building)
- copy: weekly email alerts

B: cta-ab-test (no changes)
- CTA: Follow Building
- copy: weekly building updates
